keyboards/converter/usb_usb/keymaps/chriskopher/rules.mk

```diff
@@ -16,9 +16,15 @@
 COMMAND_ENABLE = no # Not being used, and can cause conflicts with space cadet shifting
 SPACE_CADET_ENABLE = no # Not using special built in space cadet keys, using custom tap dance instead
 EXTRAKEY_ENABLE = yes # Used for audio control and system control keys
-COMBO_ENABLE = yes # Used to allow chording of keys to trigger an action
+
+# Have to disable combo feature as firmware size is too big for ble:chriskopher target, breaking CI build.
```

Comment:
This can be reverted.  CI breaking on user keymaps is okay.

Reply:
I see. Thanks for the confirmation. I will update the PR to revert changes to user keymaps.

Reply:
@drashna On a second thought. Since broken user keymaps will cause the same CI errors when new changes are introduced. Maybe it is better to merge with my temporary fixes to them to keep the CI build clean?

Reply:
Broken keymaps are likely to be pruned in the near future, so "fixing" them like this shouldn't be a concern.

Reply:
@drashna  I reverted the temporary fixes to user keymaps and this PR is ready to be reviewed and merged.